src/python/ovms_py_tensor.hpp

```diff
@@ -83,6 +85,7 @@ const std::unordered_map<std::string, py::ssize_t> bufferFormatToItemsize{
     {"e", 2},
     {"f", 4},
     {"d", 8},
+    {"s", 1},
```

Comment:
How is it 1? `BYTES` type was considered custom type since in a way it's defined in KServe, it's impossible to determine size of a single element.